rt/rs/client/src/main/java/org/apache/cxf/jaxrs/client/cache/Entry.java

```diff
@@ -64,12 +66,19 @@ public void setData(final Serializable data) {
         this.data = data;
     }
 
-    public MultivaluedMap<String, String> getHeaders() {
+    public Map<String, List<String>> getHeaders() {
```

Comment:
This an API change, perhaps best to convert back into a MultivaluedMap for backwards compatiblity

Reply:
I will do, in any case that API is only used in org.apache.cxf.jaxrs.client.cache.CacheControlClientRequestFilter where the mere Map API is really used